net/remote/remote-client.sh

```diff
@@ -37,7 +37,6 @@ snap)
 local|tar)
   PATH="$HOME"/.cargo/bin:"$PATH"
   export USE_INSTALL=1
-  export SOLANA_DEFAULT_METRICS_RATE=1
```

Comment:
Oh a couple more places too:
1. https://github.com/solana-labs/solana/blob/017c281eaf764fe8c467ebec4e5495d0404d9a3c/scripts/snap-config-to-env.sh#L12
1. https://github.com/solana-labs/solana/blob/017c281eaf764fe8c467ebec4e5495d0404d9a3c/net/remote/remote-node.sh#L49

Reply:
thanks for catching it...removed it as well